server/modules/elastic/elasticdetectionstore.go

```diff
@@ -263,6 +240,19 @@ func (store *ElasticDetectionstore) deleteDocument(ctx context.Context, index st
 		return "", err
 	}
 	defer res.Body.Close()
+
+	document := convertObjectToDocumentMap(kind, obj, store.schemaPrefix)
+	document[store.schemaPrefix+AUDIT_DOC_ID] = id
+	document[store.schemaPrefix+"kind"] = kind
+	document[store.schemaPrefix+"operation"] = "delete"
+	err = store.audit(ctx, document, id)
+	if err != nil {
+		log.WithFields(log.Fields{
+			"documentId": id,
+			"kind":       kind,
```

Comment:
This field should be `detectionKind` to avoid log ingest field type collisions with unrelated log lines.